client/app/components/queries/ScheduleDialog.jsx

```diff
@@ -206,7 +206,7 @@ export class ScheduleDialog extends React.Component {
             <h5>On time</h5>
             <div data-testid="time">
               <TimePicker
-                allowEmpty={false}
+                allowClear={false}
```

Comment:
How did you know this changed? Console error?

Reply:
Yes, you can know it in test

```
    console.error node_modules/warning/warning.js:34
      Warning: `allowEmpty` in TimePicker is deprecated. Please use `allowClear` instead.
```